akka-remote/src/main/resources/reference.conf

```diff
@@ -249,6 +249,10 @@ akka {
             advertisement-interval = 1 minute # TODO find good number as default, for benchmarks trigger immediately
           }
         }
+
+        # Remote instruments which should be initialized and used for remote messages.
+        # Refer to `akka.remote.artery.RemoteInstrument` for more information.
+        instruments = ${?akka.remote.artery.advanced.instruments} []
```

Comment:
I think that it is an absolute necessity to enable _automagic_ adding of an _instrument_ by just adding a jar to the classpath. I can see a debugging instrument for serviceability as one use of this.